sycl/source/detail/event_impl.hpp

```diff
@@ -22,6 +22,7 @@ __SYCL_INLINE_NAMESPACE(cl) {
 namespace sycl {
 class context;
 namespace detail {
+struct DepDesc;
```

Comment:
Hm, not sure how it works, there is `std::vector<DepDesc>` in the file which should require "complete" definition of the `DepDesc`. Which means this forward declaration is not needed, instead we need to somehow include file which contains definition of `DepDesc`.

Reply:
I included commands.hpp to this header